source/Ship.cpp

```diff
@@ -2925,13 +3193,23 @@ double Ship::TransferFuel(double amount, Ship *to)
 }
 
 
+// Transfer energy to another ship.
```

Comment:
Now I want a "transfer hull to another ship" as well in the case of a disabled ship.
(kidding, obviously)

Reply:
Not necessarily transfer hull but boarding a disabled ship does restore hull by the same means except not reducing hull from the current ship.